packages/web/src/components/gcds-footer/gcds-footer.tsx

```diff
@@ -66,29 +61,51 @@ export class GcdsFooter {
     return !!this.el.querySelector('[slot="list"]');
   }
 
+  private get hasContextual() {
+    return !!this.el.querySelector('[slot="contextual"]');
+  }
+
   render() {
-    const { lang, display, topHref, hasList, renderSignature } = this;
+    const { lang, display, hasList, hasContextual, renderSignature } = this;
     const govNav = I18N[lang].gov.menu;
+    const themeNav = I18N[lang].themes.menu;
     const siteNav = I18N[lang].site.menu;
 
     return (
       <Host role="contentinfo">
         <div class="top__container">
           <slot name="top"></slot>
         </div>
-
+        {hasContextual && 
+          <div class="gcds-footer__contextual">
+            <div class="contextual__container">
+              <slot name="contextual"></slot>
```

Comment:
Instead of adding a slot and trusting the user to add the proper tags (aka nav + h3 heading, ul), should we include these and only use the slot to let users add their own list items? It would ensure better accessibility since we could set up the heading as nav label etc. We could add a prop for the contextual heading?